src/Process/LayeredMaterialNodeProcessing.js

```diff
@@ -352,6 +360,15 @@ export function updateLayeredMaterialNodeElevation(context, layer, node, force)
                 return;
             }
 
+            // There's conflict between two elevation layers
```

Comment:
```js
// Do not apply the new texture if its level is < than the current one.
// This is only needed for elevation layers, because we may have several
// concurrent layers but we can only use one texture.
```

Reply:
ok, thanks